.github/workflows/json-regenerate-check.yml

```diff
@@ -31,8 +32,8 @@ jobs:
           python3 ./source/client-side-encryption/etc/generate-test.py ./source/client-side-encryption/etc/test-templates/*.template ./source/client-side-encryption/tests/legacy
           python3 ./source/client-side-operations-timeout/etc/generate-basic-tests.py ./source/client-side-operations-timeout/etc/templates ./source/client-side-operations-timeout/tests
           python3 ./source/etc/generate-handshakeError-tests.py
+          node ./source/etc/generate-crud-id-type-tests.mjs
```

Comment:
Nit: since all other scripts to generate test files are written in Python, should this one really be written in a different language?

Reply:
I wouldn't say no to php scripts :)